### PR TITLE
Add warning that library add command will be removed in a future release

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

**Comment by Fatme** [comment]
@ikoevska

**Comment by teobugslayer** [comment]
:+1:

File: docs/man_pages/lib-management/library-add.md [hunk 1]
```diff
@@ -7,21 +7,23 @@ General | `$ tns library add <Platform> <Library Path>`
 
 Adds a locally stored native library to the current project. <% if(isHtml) { %>Copies the library files to the `lib/<platform>` folder in your project and configures the platform-specific projects in `platforms/<platform>` to work with the library. Build operations might perform additional configuration changes on the platform-specific project in `platforms/<platform>`.<% } %>
 
+IMPORTANT: The `tns library add` command is deprecated and will be removed in a future release. Use the plugin set of commands instead. For more information, run `tns help plugin`.
```
Comment (ikoevska) [inline docs/man_pages/lib-management/library-add.md]: Here, we can make this more user-friendly. For the "for more information part", use context.

```
For more information, <% if(isHtml) { %>see the [plugin](/plugin.html) set of commands.<% } %><% if(isConsole) { %>run `tns help plugin`.<% } %>
```

Comment (ikoevska) [inline docs/man_pages/lib-management/library-add.md]: Also, it seems that we're probably deprecating the entire library set of commands. Shouldn't we apply this message to the library command as well? If we decide to do this Make the message "The `tns library` set of commands is deprecated and..."

Comment (ikoevska) [inline docs/man_pages/lib-management/library-add.md]: And make this "Use the `tns plugin` set of commands."

File: docs/man_pages/lib-management/library.md [hunk 2]
```diff
@@ -7,11 +7,13 @@ General | `$ tns library <Command>`
 
 Lets you manage third-party native libraries in your project. You must run the `library` command with a command extension.
 
+IMPORTANT: The `tns library` set of commands are deprecated and will be removed in a future release. Use the `tns plugin` set of commands instead. For more information, <% if(isHtml) { %>see the [plugin](plugin.html) set of commands.<% } %><% if(isConsole) { %>run `tns help plugin`.<% } %>.
```
Comment (ikoevska) [inline docs/man_pages/lib-management/library.md]: Hmmm, make this "The `tns library` command is deprecated" perhaps - sorry about that.

**Comment by ikoevska** [comment]
:+1:

**Pull request merged**